packages/styles/scss/internal/content-block/_content-block.scss

```diff
@@ -12,15 +12,25 @@
 
 @mixin content-block {
   .#{$prefix}--content-block {
-    padding-top: $carbon--layout-03;
-    padding-bottom: $carbon--layout-05;
-    @include carbon--breakpoint('md') {
-      padding-top: $carbon--layout-05;
-      padding-bottom: $carbon--layout-06;
+    padding-top: $layout-05;
+    padding-bottom: $layout-07;
+
+    @include carbon--breakpoint-down('lg') {
+      padding-bottom: $layout-03;
+
+      &:not(.#{$prefix}--content-block_has-linkList) {
```

Comment:
`has-linkList` might not be the best name as the `aside` can have any type of content.